src/test/java/ancaMarian/tests/tema8/tests/DynamicButtonsSimpleTest.java

```diff
@@ -0,0 +1,26 @@
+package ancaMarian.tests.tema8.tests;
+
+import ancaMarian.tests.tema8.pageObjects.DynamicButtonsSimplePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class DynamicButtonsSimpleTest extends BaseTest {
+
+    @Test(groups = {"mobile", "desktop"})
+    public void dynamicButtonsSimplePageTest() {
+        driver.get("https://testpages.herokuapp.com/styled/dynamic-buttons-simple.html");
+        DynamicButtonsSimplePage dynamicButtonsSimplePage = new DynamicButtonsSimplePage(driver);
+        dynamicButtonsSimplePage.waitForStartButtonToBeVisible();
+        dynamicButtonsSimplePage.clickOnStartButton();
+        dynamicButtonsSimplePage.waitForOneButtonToBeVisible();
+        dynamicButtonsSimplePage.clickOnOneButton();
+        dynamicButtonsSimplePage.waitForWaitMessageToDisappear();
+        dynamicButtonsSimplePage.clickOnTwoButton();
+        dynamicButtonsSimplePage.waitForWaitMessageToDisappear();
+        dynamicButtonsSimplePage.clickOnThreeButton();
+        WebElement confirmationMessage = driver.findElement(By.id("buttonmessage"));
```

Comment:
Move this element in page object class and create a method to get it's text to use it in the assert. Thanks!